apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/mapEntryExpectations.kt

```diff
@@ -9,6 +9,8 @@ import ch.tutteli.atrium.logic.*
  *
  * @return an [Expect] for the subject of `this` expectation.
  *
+ * @sample ch.tutteli.atrium.api.infix.en_GB.samples.MapEntryExpectationSamples.toEqualKeyValue
```

Comment:
same same for other methods, also add the `@sample` tag to the `key` and `value` overloads. You can check if it works by holding ALT + clicking with your middle mouse on the method name